.eslintrc.yml

```diff
@@ -3,41 +3,30 @@
     modules: true
   env:
     browser: false
-    es6: true
```

Comment:
Recommend adding `es6` and `mocha` back in. You'll also want to add the following:

```yml
globals:
  should: true
```

And then just above `no-undefined: 2`, add `no-undef: 2`